### DISCUSSION TITLE
Discussion: Release notes custom text

### DISCUSSION THREAD

**Discussion by olivermt** [basecamp/kamal#269] [General]
Hi,

Just a quick comment is I wonder if release notes should be a tiny bit more structured.
I hadn't pinned mrsk on github actions, so when 0.12 was released my healthchecks stopped working (due to missing curl on the image).
Nothing in the release info about this, just new info in the readme.

Maybe PRs that include breaking changes should also have some kind of text that could optionally go into the release notes? It is a manual process so I understand if there is no will or time budget to do this, but I think that if a version bump kills an existing deploy there should probably at least be a mention about it in the github release :) 

Thanks for a great tool!

**Comment by 34code** [comment]
how did you fix the missing `curl` problem for healthchecks?

**Reply by 34code** [reply]
If i understand correctly, its due to `curl` not being in the healthcheck image, correct?

**Reply by olivermt** [reply]
correct!
